playbooks/fdp_update.yml

```diff
@@ -0,0 +1,168 @@
+
+- name: Update OpenStack packages across all layers
+  hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: true
+  vars:
+    # Control flags for what to update (can be overridden via -e)
+    cifmw_fdp_update_container_images_update_edpm_images: true
+    cifmw_fdp_update_container_images_update_edpm_host_packages: true
+
+  pre_tasks:
+    - name: Early playbook stop if disabled
+      when:
+        - not cifmw_fdp_update_enabled | default(false) | bool
+      ansible.builtin.meta: end_play
+
+    - name: Validate required variables are set
+      ansible.builtin.assert:
+        that:
+          - cifmw_fdp_update_container_images_target_package is defined
+          - cifmw_fdp_update_container_images_target_package | length > 0
+          - cifmw_fdp_update_container_images_repo_baseurl is defined
+          - cifmw_fdp_update_container_images_repo_baseurl | length > 0
+        fail_msg: |
+          Required variables are missing!
+
+          You must set:
+            - cifmw_fdp_update_container_images_target_package: Name of the RPM package to update
+            - cifmw_fdp_update_container_images_repo_baseurl: Repository base URL containing the updated package
+
+          Example:
+            ansible-playbook playbooks/fdp_update.yml \
+              -e cifmw_fdp_update_container_images_target_package=ovn24.03 \
+              -e cifmw_fdp_update_container_images_repo_baseurl=http://example.com/repo/
+        success_msg: "Required variables validated successfully"
+
+    - name: Display update configuration
+      ansible.builtin.debug:
+        msg:
+          - "=============================================="
+          - "OpenStack Package Update Configuration"
+          - "=============================================="
+          - "Target Package: {{ cifmw_fdp_update_container_images_target_package }}"
+          - "Repository: {{ cifmw_fdp_update_container_images_repo_baseurl }}"
+          - ""
+          - "Update Control Flags:"
+          - "  Control Plane Images: {{ cifmw_fdp_update_container_images_update_control_plane_images | default(true) }}"
+          - "  EDPM Container Images: {{ cifmw_fdp_update_container_images_update_edpm_images | default(true) }}"
+          - "  EDPM Host Packages: {{ cifmw_fdp_update_container_images_update_edpm_host_packages | default(true) }}"
+          - ""
+          - "Namespace: {{ cifmw_fdp_update_container_images_namespace | default('openstack') }}"
+          - "=============================================="
+
+    - name: Setup hypervisor firewall for registry access
+      become: true
+      when: cifmw_fdp_update_setup_hypervisor_firewall | default(true) | bool
+      block:
+        - name: Allow traffic from osp_trunk to ocpbm (compute -> registry)
+          ansible.builtin.command: # noqa: command-instead-of-module
+            cmd: iptables -I FORWARD -i osp_trunk -o ocpbm -j ACCEPT
```

Comment:
~~are you sure to do that via iptables and not `firewall-cmd`?~~
From the other side, why not module?